framework/include/cppmicroservices/Bundle.h

```diff
@@ -398,6 +425,9 @@ class US_Framework_EXPORT Bundle
    * @return The value of the requested property, or an empty string
    *         if the property is undefined.
    *
+   * @deprecated Since 3.0, use GetHeaders() or BundleContext::GetProperty
```

Comment:
Yes, I missed to mention this in the initial comment. I did not do this on purpose just yet. Are you ready to handle these kind of compiler warnings? Some other projects I worked with had troubles suppressing third party warnings, especially when compiling with `-Werror` (warnings as errors). I just want to make sure this is okay for you.